config/cors.js

```diff
@@ -28,54 +28,55 @@
-  ***************************************************************************/
+    /***************************************************************************
+    *                                                                          *
+    * Allow CORS on all routes by default? If not, you must enable CORS on a   *
+    * per-route basis by either adding a "cors" configuration object to the    *
+    * route config, or setting "cors:true" in the route config to use the      *
+    * default settings below.                                                  *
+    *                                                                          *
+    ***************************************************************************/
 
-  allRoutes: true,
+    allRoutes: true,
 
-  /***************************************************************************
-  *                                                                          *
-  * Which domains which are allowed CORS access? This can be a               *
-  * comma-delimited list of hosts (beginning with http:// or https://) or    *
-  * "*" to allow all domains CORS access.                                    *
-  *                                                                          *
-  ***************************************************************************/
+    /***************************************************************************
+    *                                                                          *
+    * Which domains which are allowed CORS access? This can be a               *
+    * comma-delimited list of hosts (beginning with http:// or https://) or    *
+    * "*" to allow all domains CORS access.                                    *
+    *                                                                          *
+    ***************************************************************************/
 
-  origin: '*',
-  // process.env.CORS_ORIGIN || 'https://www.hylo.com',
+    allowOrigins: '*',
+    // process.env.CORS_ORIGIN || 'https://www.hylo.com',
 
-  /***************************************************************************
-  *                                                                          *
-  * Allow cookies to be shared for CORS requests?                            *
-  *                                                                          *
-  ***************************************************************************/
+    /***************************************************************************
+    *                                                                          *
+    * Allow cookies to be shared for CORS requests?                            *
+    *                                                                          *
+    ***************************************************************************/
 
-  credentials: true,
+    allowCredentials: false,
```

Comment:
Curious what you learned here and whether this all works ok through the proxy. I think if it's working on staging though it works on production in this case...